portal_sale_hide_payment_terms/views/sale_portal_templates.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template
+        id="sale_order_portal_content"
+        inherit_id="sale.sale_order_portal_content"
+    >
+        <xpath expr="//div/section[4]" position="replace" />
```

Comment:
```suggestion
        <xpath expr="//section[@t-if='sale_order.payment_term_id']" position="attributes">
            <attribute name="style">display: none;</attribute>
        </xpath>
```
Would this work, or do we need to do 'replace' to prevent the payment term from being added to the index (on the left side)?

Reply:
I think we must replace to prevent payment term from left side. I tried with your suggested code but it doesn't prevent payment term from left side.
![Screenshot from 2022-12-01 12-59-23](https://user-images.githubusercontent.com/45355704/204981909-a4f4a7f6-8c73-4d54-8f0b-79603a1a6d88.png)

Reply:
OK. Please still switch the xpath expr.  It is not easy to tell which part is being replaced with `//div/section[4]`, and it also adds extra risk to specify the position number as it could change even though the chances are slim.